### ISSUE TITLE
potpourri3d laplacian and mass matrix not same as torch3d

### ISSUE THREAD

**Issue by BarakeelFanseuKamhoua** [facebookresearch/pytorch3d#1984]
## torch3d
      L, _areas = p3d.ops.cot_laplacian(verts, faces, eps=1e-8)
      _areas = torch.diag(_areas.squeeze())
## potpurri
      verts_np, faces_np = verts.detach().cpu().numpy(), faces.detach().cpu().numpy()
      L = pp3d.cotan_laplacian(verts_np, faces_np, denom_eps=1e-10)
      _areas = pp3d.vertex_areas(verts_np, faces_np)
      _areas += 1e-8 * np.mean(_areas)
      L = torch.from_numpy(L.toarray()).to(self.device).float()
      _areas = torch.from_numpy(_areas).to(self.device).float() 
      _areas = torch.diag(_areas.squeeze())
